prow/lib.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+# Copyright 2018 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function setup_and_export_git_sha() {
+  if [[ -n "${CI:-}" ]]; then
+    if [[ "${CI:-}" == 'bootstrap' ]]; then
+      # TODO: Remove after update to pod-utils
+
+      # Make sure we are in the right directory
+      # Test harness will checkout code to directory $GOPATH/src/github.com/istio/istio
+      # but we depend on being at path $GOPATH/src/istio.io/istio for imports
+      if [[ ! $PWD = ${GOPATH}/src/istio.io/istio ]]; then
```

Comment:
not sure if this will work as is or needs to be `/installer`. the test isn't running here so hard to tell

Reply:
I am not using most of this prow/lib.sh code :)